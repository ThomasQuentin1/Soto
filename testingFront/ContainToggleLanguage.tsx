import React from "react";
import ToggleLanguage from "../components/settings/ToggleLanguage";
import { useTranslation } from "react-i18next";

const ContainToggleLanguage = () => {
  const { t, i18n } = useTranslation();
  i18n;
  return <ToggleLanguage t={t}></ToggleLanguage>;
};

export default ContainToggleLanguage;
