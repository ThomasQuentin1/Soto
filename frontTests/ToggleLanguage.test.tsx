import React from 'react';
import renderer from 'react-test-renderer';
import ContainToggleLanguage from './ContainToggleLanguage';

test('ToggleLanguague is the components that display the button which change the language', () => {
    const component = renderer.create(
        <ContainToggleLanguage></ContainToggleLanguage>
    )

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});