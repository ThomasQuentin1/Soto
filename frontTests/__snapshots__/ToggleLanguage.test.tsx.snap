// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToggleLanguague is the components that display the button which change the language 1`] = `
<div>
  <button
    className="MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root css-zcbmsk-MuiButtonBase-root-MuiButton-root"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onContextMenu={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    Changer pour anglais
  </button>
</div>
`;
